lib/LocomotiveScroll.context.tsx

```diff
@@ -68,11 +74,31 @@ export function LocomotiveScrollProvider({
 
   useEffect(
     () => {
-      LocomotiveScrollRef.current?.update()
+      if (!LocomotiveScrollRef.current) {
+        return
+      }
+
+      LocomotiveScrollRef.current.update()
+
+      if (onUpdate) {
+        onUpdate(LocomotiveScrollRef.current)
+      }
     },
     watch ? [...watch, height] : [height]
```

Comment:
Would like to provide logging if location is provided in both of the `watch` and `location` deps list